src/k8s/components/precise-code-intel/documentation.md

```diff
@@ -0,0 +1,108 @@
+# Table of Contents
```

Comment:
Please create a doc.dhall file to test to see if your with statements documented here are correct: https://github.com/sourcegraph/deploy-sourcegraph-core/blob/main/src/k8s/components/syntect-server/doc.dhall

Reply:
You can also look at `k8s-configuration-defaults.yaml` (generated via `just build`) to see what your user interface looks like for reference

Reply:
And, `dhall diff '(./src/k8s/components/doc.dhall).precide-code-intel' '(./src/k8s/pipeline.dhall).precise-code-intel' will show if your with statements are actually being respected. (You can also point ds-to-dhall at the doc.dhall file)